prime-router/docs/features/0005-managing-translation-schemas.md

```diff
@@ -0,0 +1,793 @@
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+    <details>
+      <summary>Production</summary>
+
+        ```
+            /local-azure-container
+                /fhir_transforms
+                    validating.txt
+                    previous-valid-2023-03-29T23:53:00.488Z.txt
+                    previous-previous-valid-2023-02-25T23:53:00.488Z.txt
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+11. The azure function for validation runs successfully
+    <details>
+      <summary>Local</summary>
+
+        ```
+            /local-azure-container
+                /fhir_transforms
+                    valid-2023-03-29T23:53:00.488Z.txt
+                    previous-valid-2023-02-25T23:53:00.488Z.txt
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+    <details>
+      <summary>Staging</summary>
+
+        ```
+            /local-azure-container
+                /fhir_transforms
+                    valid-2023-04-02T23:53:00.488Z.txt
+                    previous-valid-2023-03-29T23:53:00.488Z.txt
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+    <details>
+      <summary>Production</summary>
+
+        ```
+            /local-azure-container
+                /fhir_transforms
+                    valid-2023-04-03T23:53:00.488Z.txt
+                    previous-valid-2023-03-29T23:53:00.488Z.txt
+                    /default
+                        input.fhir
+                        output.fhir
+                        default-sender-transform.yml
+                    /Flexion
+                        flexion-sender-transform.yml
+                        input.fhir
+                        output.fhir
+        ```
+    </details>
+
+#### Validation
+
+The initial validation will mimic how the current integration tests work.  Each translation schema will:
```

Comment:
The current integration tests allow you test the whole integration from sender to receiver. It seems like this new set-up will only allow you test one piece of the integration(FHIR-> HL7, HL7->FHIR, and FHIR->FHIR). It would be missing the ability to test HL7 -> FHIR -> FHIR -> HL7 and FHIR -> FHIR -> HL7. This helps us ensure that making a change to a sender schema doesn't break the receiver schemas or identify any changes we need to make on the receiver side.

Reply:
Summarizing  @victor-chaparro and I's conversation

- Specifically discussed the simple report sender transform that subtracts 15 minutes, but this is only applicable for CA.  This is an example where a sender transform could break downstream receivers
- However, this is actually a current problem; there is a HL7->FHIR->FHIR-HL7 integration test for California that would catch any issues that a sender transform would cause for CA, but there is no check that an additional receiver had been setup to receive data
  - The integration tests mitigate this problem, but still leaves a gap
- The root of the problem is overall unclear guidelines on when a change should be an update to a sender transform or receiver transform
  - New ticket captures the work to create documentation that will explain: #10435